### PR TITLE
Issue#419

### PR DESCRIPTION
**Pull request by daminichopra**
## Proposed changes

Updated pagination.md 

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [CLA][cla]
- [ ] I have added tests that prove my fix is effective or that my
      feature works
- [ x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

Updated for Issue#419

[cla]: https://fiware.github.io/contribution-requirements/individual-cla.pdf
    "FIWARE Individual Contributor License Agreement"
[contrib]: https://github.com/smartsdk/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"

### CODE REVIEW

**Comment by c0c0n3** [comment]
@daminichopra thanks for this. if you could please sign the CLA, thx!!

File: src/reporter/delete.py [hunk 1]
```diff
@@ -34,11 +34,19 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table:
```
Comment (c0c0n3) [inline src/reporter/delete.py:37]: it doesn't look like this change set belongs in this PR? it looks like it's coming from #455?

- https://github.com/orchestracities/ngsi-timeseries-api/pull/455/files

Comment (c0c0n3) [reply src/reporter/delete.py:37]: should we rollback changes to this file? or merge #455 and then rebase?

**Review by c0c0n3 — CHANGES_REQUESTED** [review]

**Comment by daminichopra** [comment]
> @daminichopra thanks for this. if you could please sign the CLA, thx!!

Hi @c0c0n3 , Now confirming the signing process inside the company, so please wait for a while. Thank you

**Comment by KazuhitoSuda-NEC** [comment]
Dear @chicco785, @c0c0n3

Your CLA file seems to be a template file provided by the FIWARE Foundation.
Could you update it for your product and put it in this repository, please?

Best regards,

**Comment by c0c0n3** [comment]
Hello @KazuhitoSuda-NEC

> Your CLA file seems to be a template file provided by the FIWARE Foundation.

Correct!

> Could you update it for your product and put it in this repository, please?

It's already in our repo:

- https://github.com/orchestracities/ngsi-timeseries-api/blob/master/.github/pull_request_template.md

what kind of updates does it need? thx!

**Comment by KazuhitoSuda-NEC** [comment]
Thank you for your reply.

There is a link of `individual-cla.pdf` file in `pull_request_template.md`.
It is outside the repository. It seems that the file is not for QuantumLeap.

- https://fiware.github.io/contribution-requirements/individual-cla.pdf

**Comment by c0c0n3** [comment]
@KazuhitoSuda-NEC thanks for pointing this out. I'm actually not sure myself what to do and have opened #479 about it. If we could please move the CLA conversation over there, thanks a stack!!

**Comment by chicco785** [comment]
@daminichopra can you rebase and trigger a rebuild?

`git rebase -i master`
`git push --force`

**Comment by daminichopra** [comment]
> @daminichopra can you rebase and trigger a rebuild?
> 
> `git rebase -i master`
> `git push --force`

Hi @chicco785 , I have rebased the PR. Thank you

**Comment by daminichopra** [comment]
Hi @c0c0n3 @chicco785 ,  I have signed the CLA for this PR.

**Comment by daminichopra** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by chicco785** [comment]
@daminichopra thanks for signing, there are few markdown issues left to be fixed:

```
docs/manuals/user/pagination.md:1 MD041/first-line-heading/first-line-h1 First line in file should be a top level heading [Context: "## Pagination"]
docs/manuals/user/pagination.md:3:81 MD013/line-length Line length [Expected: 80; Actual: 161]
docs/manuals/user/pagination.md:5:81 MD013/line-length Line length [Expected: 80; Actual: 87]
docs/manuals/user/pagination.md:8:81 MD013/line-length Line length [Expected: 80; Actual: 118]
docs/manuals/user/pagination.md:10 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
docs/manuals/user/pagination.md:14 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
docs/manuals/user/pagination.md:24:81 MD013/line-length Line length [Expected: 80; Actual: 96]
```

**Comment by daminichopra** [comment]
> @daminichopra thanks for signing, there are few markdown issues left to be fixed:
> 
> ```
> docs/manuals/user/pagination.md:1 MD041/first-line-heading/first-line-h1 First line in file should be a top level heading [Context: "## Pagination"]
> docs/manuals/user/pagination.md:3:81 MD013/line-length Line length [Expected: 80; Actual: 161]
> docs/manuals/user/pagination.md:5:81 MD013/line-length Line length [Expected: 80; Actual: 87]
> docs/manuals/user/pagination.md:8:81 MD013/line-length Line length [Expected: 80; Actual: 118]
> docs/manuals/user/pagination.md:10 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
> docs/manuals/user/pagination.md:14 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
> docs/manuals/user/pagination.md:24:81 MD013/line-length Line length [Expected: 80; Actual: 96]
> ```

Hi @chicco785 , Updated markdown issues in pagination.md.

**Comment by daminichopra** [comment]
Hi @c0c0n3 @chicco785 , Please close this PR if it looks good.

File: src/reporter/delete.py (hunk 1 above)
Comment (chicco785) [reply src/reporter/delete.py:37]: can you remove changes to this file? thx

**Comment by daminichopra** [comment]
Gentle Reminder!!

File: src/reporter/delete.py (hunk 1 above)
Comment (chicco785) [reply src/reporter/delete.py:37]: @daminichopra see my comment above, if you take out this commit from the pr, we will merge it, thx!
if you have trouble doing so, i can open a new pr with the pagination part for you.

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: src/reporter/delete.py (hunk 1 above)
Comment (daminichopra) [reply src/reporter/delete.py:37]: @chicco785 , Yes my PR has some issues with master branch so not able to remove these commits.  Thanks for suggestion  :)

**Pull request closed without merging**